tidb-in-kubernetes: update helm content in aws deployment doc (#1984)
pingcap/docs-cn: dev/tidb-in-kubernetes/deploy/aws-eks.md

  monitor_instance_type         = "t2.xlarge"
  # The version of tidb-cluster helm chart
  tidb_cluster_chart_version    = "v1.0.0"
  # Decides whether or not to create the tidb-cluster helm release.

>   # If this variable set to false, you have to

Reviewer: ```suggestion
  # If this variable is set to false, you have to
```

  monitor_instance_type         = "t2.xlarge"
  # The version of tidb-cluster helm chart
  tidb_cluster_chart_version    = "v1.0.0"

>   # Wether creating the tidb-cluster helm release, if this variable set to false, you have to

Reviewer: ```suggestion
  # Decides whether or not to create the tidb-cluster helm release.
  # If this variable set to false, you have to
```

  # install the helm release manually

>   create_tidb_cluster_release   = true

Reviewer 2: Where is this variable used to control the installation of tidb cluster?

Author: https://github.com/pingcap/tidb-operator/pull/1062 fix this

最后，只要移除 `tidb-cluster` 模块调用，对应的 TiDB 集群就会被销毁，EC2 资源也会随之释放。

## 仅管理基础设施

> 假如你更希望用 Helm 来直接管理 TiDB 集群，那么你可以调整 Terraform 脚本的配置，控制 Terraform 脚本只创建 Kubernetes 集群和工作节点等基础设施。该操作可以通过修改 `clusters.tf` 中 TiDB 集群的 `create_tidb_cluster_release` 配置项实现：

Reviewer 2: ```suggestion
假如你更希望用 Helm 来直接管理 TiDB 集群，那么你可以调整 Terraform 脚本的配置，控制 Terraform 脚本只创建 Kubernetes 集群和工作节点等基础设施以及 TiDB Operator。该操作可以通过修改 `clusters.tf` 中 TiDB 集群的 `create_tidb_cluster_release` 配置项实现：
```

> 假如你更希望用 Helm 来直接管理 TiDB 集群，那么你可以调整 Terraform 脚本的配置，控制 Terraform 脚本只创建 Kubernetes 集群和工作节点等基础设施。该操作可以通过修改 `clusters.tf` 中 TiDB 集群的 `create_tidb_cluster_release` 配置项实现：

Reviewer: ```suggestion
如果你更想用 Helm 来直接管理 TiDB 集群，可调整 Terraform 脚本的配置，使脚本只创建 Kubernetes 集群和工作节点等基础设施。要实现这一操作，可修改 `clusters.tf` 中 TiDB 集群的 `create_tidb_cluster_release` 配置项：
```

```hcl
module "default-cluster" {
  ...
  create_tidb_cluster_release = false
}
```

> 如上所示，将 `create_tidb_cluster_release` 修改为 `false` 后，Terraform 脚本就不会创建和修改 TiDB 集群对应的 helm release。此时，你需要使用 Helm 独立管理 TiDB 集群。

Reviewer 2: ```suggestion
如上所示，将 `create_tidb_cluster_release` 修改为 `false` 后，Terraform 脚本就不会创建和修改 TiDB 集群。此时，你需要使用 Helm 独立管理 TiDB 集群。
```

> 如上所示，将 `create_tidb_cluster_release` 修改为 `false` 后，Terraform 脚本就不会创建和修改 TiDB 集群对应的 helm release。此时，你需要使用 Helm 独立管理 TiDB 集群。

Reviewer: ```suggestion
如上所示，将 `create_tidb_cluster_release` 修改为 `false` 后，Terraform 脚本将不再创建和修改 TiDB 集群对应的 helm release。此时，你需要使用 Helm 来独立管理 TiDB 集群。
```

> **注意：**
>

> > 在已经部署的集群上将 `create_tidb_cluster_release` 调整为 `false` 会导致已安装的 Helm Release 被删除，对应的 TiDB 集群对象也会随之被删除。

Reviewer 2: ```suggestion
> 在已经部署的集群上将 `create_tidb_cluster_release` 调整为 `false` 会导致已安装的 TiDB 集群被删除，对应的 TiDB 集群对象也会随之被删除。
```

最后，只要移除 `tidb-cluster` 模块调用，对应的 TiDB 集群就会被销毁，EC2 资源也会随之释放。

## 仅管理基础设施

> 假如你更希望用 Helm 来直接管理 TiDB 集群，那么你可以调整 Terraform 脚本的配置，控制 Terraform 脚本只创建 Kubernetes 集群和工作节点等基础设施以及 TiDB Operator。该操作可以通过修改 `clusters.tf` 中 TiDB 集群的 `create_tidb_cluster_release` 配置项实现：

Reviewer: This Chinese sentence is not so readable and clear. “你” and the English (sentence) style might look strange to readers.

```hcl
module "default-cluster" {
  ...
  create_tidb_cluster_release = false
}
```

> 如上所示，将 `create_tidb_cluster_release` 修改为 `false` 后，Terraform 脚本就不会创建和修改 TiDB 集群。此时，你需要使用 Helm 来独立管理 TiDB 集群。

Reviewer: `就不会` also looks a little weird and informal.

> **注意：**
>

> > 在已经部署的集群上将 `create_tidb_cluster_release` 调整为 `false` 会导致已安装的 TiDB 集群被删除，对应的 TiDB 集群对象也会随之被删除。

Reviewer 2: ```suggestion
> 在已经部署的集群上将 `create_tidb_cluster_release` 调整为 `false` 会导致已安装的 TiDB 集群被删除。
```
Is it better to remove the "对应的 TiDB 集群对象也会随之被删除"?
It may confuse users what is the "TiDB 集群对象".

最后，只要移除 `tidb-cluster` 模块调用，对应的 TiDB 集群就会被销毁，EC2 资源也会随之释放。

## 仅管理基础设施

通过调整配置，你可以控制 Terraform 脚本只创建 Kubernetes 集群和 TiDB Operator。操作步骤如下：

> * 修改 `clusters.tf` 中 TiDB 集群的 `create_tidb_cluster_release` 配置项：

Reviewer 2: Does the list only have one item?

Author: Yes
